src/Facebook/FacebookClient.php

```diff
@@ -203,32 +193,29 @@ public function sendRequest(FacebookRequest $request)
 
         list($url, $method, $headers, $body) = $this->prepareRequestMessage($request);
 
-        // Since file uploads can take a while, we need to give more time for uploads
-        $timeOut = static::DEFAULT_REQUEST_TIMEOUT;
-        if ($request->containsFileUploads()) {
-            $timeOut = static::DEFAULT_FILE_UPLOAD_REQUEST_TIMEOUT;
-        } elseif ($request->containsVideoUploads()) {
-            $timeOut = static::DEFAULT_VIDEO_UPLOAD_REQUEST_TIMEOUT;
-        }
-
-        // Should throw `FacebookSDKException` exception on HTTP client error.
-        // Don't catch to allow it to bubble up.
-        $rawResponse = $this->httpClientHandler->send($url, $method, $body, $headers, $timeOut);
+        $psr7Request = MessageFactoryDiscovery::find()->createRequest($method, $url, $headers, $body);
```

Comment:
I'm not sure about this change, it doesn't add much value, and for me having the pair `$psr7Request` and `$psr7Response` make things explicit and clearer.
Anyway, no strong requirement for this, i'll accept it as is. (it can be change later haha)